### PR TITLE
fix(Puppeteer): #waitForText - Cannot read property 'innerText' of null

### PR DESCRIPTION
**Pull request by Vorobeyko**
## Motivation/Description of the PR

Applicable helpers:

- [ ] Webdriver
- [x] Puppeteer
- [ ] Nightmare
- [ ] REST
- [ ] Appium
- [ ] Protractor
- [ ] TestCafe

- Description of this PR, which problem it solves
- Resolves #issueId (if applicable).

## Type of change

- [ ] Breaking changes
- [ ] New functionality
- [x] Bug fix
- [ ] Documentation changes/updates
- [ ] Hot fix
- [ ] Markdown files fix - not related to source code

## Checklist:

- [x] Tests have been added
- [ ] Documentation has been added (Run `npm run docs`)
- [x] Lint checking (Run `npm run lint`)
- [x] Local tests are passed (Run `npm test`)

### CODE REVIEW

File: test/helper/Puppeteer_test.js [hunk 1]
```diff
@@ -649,6 +649,12 @@ describe('Puppeteer', function () {
       .then(() => I.see('button was clicked', '#message')));
   });
 
+  describe('#waitForText', () => {
+    it('should wait for text after load body', () => I.amOnPage('/redirect_long')
+      .then(() => I.waitForText('Hi there and greetings!', 5))
+    );
```
Comment (DavertMik) [inline test/helper/Puppeteer_test.js]: Codacy Issue found: [Unexpected newline before ')'. (function-paren-newline)](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4937810)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4937810&bid=16327480)

File: test/helper/Puppeteer_test.js [hunk 2]
```diff
@@ -649,6 +649,12 @@ describe('Puppeteer', function () {
       .then(() => I.see('button was clicked', '#message')));
   });
 
+  describe('#waitForText', () => {
+    it('should wait for text after load body', () => I.amOnPage('/redirect_long')
+      .then(() => I.waitForText('Hi there and greetings!', 5))
```
Comment (DavertMik) [inline test/helper/Puppeteer_test.js]: Codacy Issue found: [Missing trailing comma.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4937810)

File: test/helper/Puppeteer_test.js [hunk 3]
```diff
@@ -649,6 +649,12 @@ describe('Puppeteer', function () {
...
+    );
+  })
```
Comment (DavertMik) [inline test/helper/Puppeteer_test.js]: Codacy Issue found: [Missing semicolon.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4937810)

File: test/helper/Puppeteer_test.js [hunk 4]
```diff
@@ -649,6 +649,11 @@ describe('Puppeteer', function () {
       .then(() => I.see('button was clicked', '#message')));
   });
 
+  describe('#waitForText', () => {
+    it('should wait for text after load body', () => I.amOnPage('/redirect_long')
+      .then(() => I.waitForText('Hi there and greetings!', 5)));
+  });
```
Comment (PeterNgTr) [inline test/helper/Puppeteer_test.js]: could we write tests as async/await?

Comment (Vorobeyko) [reply test/helper/Puppeteer_test.js]: Yes, but why?)

Comment (PeterNgTr) [reply test/helper/Puppeteer_test.js]: better to read :)

Comment (Vorobeyko) [reply test/helper/Puppeteer_test.js]: ok)

**Pull request merged**